lecture_2/homework/.rubocop.yml

```diff
@@ -0,0 +1,1069 @@
+AllCops:
```

Comment:
To jest chyba domyślny config rubocopa 👀 Nie musisz dodawać go do projektu  - rubocop ma go w swoim kodzie i sam go ładuje. W pliku `.rubocop.yml` dopisujesz tylko te reguły które chcesz zmienić